schema-engine/sql-migration-tests/src/multi_engine_test_api.rs

```diff
@@ -210,6 +210,11 @@ impl TestApi {
         self.args.tags()
     }
 
+    /// Return the provider for the datasource block in the schema.
+    pub fn provider(&self) -> &str {
```

Comment:
Used in `connecting_to_a_postgres_database_with_missing_schema_creates_it` test.